src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/services/UserService.kt

```diff
@@ -89,5 +98,4 @@ class UserService(
     return null
 
```

Comment:
you didn't write this part but while editing stuff here might as well remove this extra return?